### PR TITLE
feat: add home path in breadcrumb

### LINKED ISSUE #846: [FEATURE] Breadcrumbs include a home-button
### Description
Breadcrumbs include a home button on pages that are not the main page.

### Acceptance criteria

- Home Button is at the first position
- Home button is hidden on Start page but visible anywhere else
- Text editable through i18n

Below my implementation without i18n, might be useful.
```      const getBreadcrumbs = computed(() => {
          const breadcrumbs = Object.values(getBreadcrumbsObject.value).map((breadcrumb) => ({
            text: breadcrumb.name,
            link: vm.$i18n.path(breadcrumb.path)
          }))
          if (breadcrumbs.length > 0) {
            breadcrumbs.unshift({
              text: 'Home',
              link: vm.$i18n.path('/')
            })
          }
        return breadcrumbs
        }
      )
```

### PR DESCRIPTION
**Pull request by luklus**
## Changes
closes #846

Zrzut ekranu z 2020-06-30 14-26-18

### Checklist

- [x] I followed [contributing](https://github.com/DivanteLtd/shopware-pwa/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by luklus** [comment]
@akrajna  there is no breadcrumbs on product detail page, there is no breadcrumbs on 'my account' page - https://github.com/DivanteLtd/shopware-pwa/projects/15#card-41063261

**Comment by luklus** [comment]
@akrajna breadcrumbs in checkout depends on from which page user enter the checkout e.g - no breadcrumbs on checkout page

**Comment by akrajna** [comment]
Works fine. As discussed there are breadcrumbs on categories page and checkout, we will work on breadcrumbs on product page.

File: packages/default-theme/layouts/default.vue [hunk 1]
```diff
@@ -63,22 +68,38 @@ export default {
...
-    )
+          route: {
+            link: root.$i18n.path(breadcrumb.path),
+          },
+        })
+      )
+
+      if (breadcrumbs.length > 0) {
+        breadcrumbs.unshift({
+          text: "Home",
```
Comment (patzick) [inline packages/default-theme/layouts/default.vue:84]: This does not meet Acceptance criterium:
`Text editable through i18n`

**Pull request merged**